css/main.css

```diff
@@ -54,7 +54,8 @@ nav .logo {
 nav .user-greet {
   display: none;
   font-size: 20px;
-  margin-top: 10px;
+  margin-top: 20px 30px 0 0;
```

Comment:
Could you please look into this...
It should be `margin`

https://developer.mozilla.org/en-US/docs/Web/CSS/margin